web/wp-content/themes/bumblebee/inc/ads.php

```diff
@@ -73,37 +71,3 @@ function( $key ) use ( $ad_options ) {
 }
 
 
-/**
- * Prefix every ad slot name with the Ad Unit Path
- *
- * @param mixed[] $ad_options   Ad rendering options.
- */
-function bumblebee_add_slot_name_prefix( $ad_options ) {
-	// @todo: make this filterable and read proper tokens for ad unit 2, 3 and 4.
-	$ad_options['slot-name'] = '/6178/fhm_desktop/homepage/homepage/' . $ad_options['slot-name'];
```

Comment:
`/6178/fhm_desktop` (property, site and platform) seem to be missing in new code.

Reply:
property, site and platform comes from global ad targeting localized data so we can grab from there.